### ISSUE TITLE
Automated installer for Debian 11

### ISSUE THREAD

**Issue by jakeogh** [drogenlied/linux-gpib-dkms#1]
Hello!

Thank you for creating this, I ended up making an automated installer based on the gist: https://gist.github.com/jonathanschilling/07defddc272fe35c7412d51dffa0bb6f

https://github.com/jakeogh/linux-gpib-installer

-Justin

**Comment by drogenlied** [comment]
@jakeogh Hi!
If you or someone you know is still interested in more convenience, I managed to package more or less all of linux-gpib, check out drogenlied/gpib.

I started out with GauiStori/gpib though not that much is left of their original work and the repository is not usable with git-buildpackage anymore.

On my machine with Debian unstable, debuild -i -us- uc -b runs and only has 4 lintian wanings, but the general strategy for building the package is a total hack that abuses debhelper.

Maybe I will look into better debug symbol packages next, I have given up on the php and guile bindings.

--
Jan
